src/Aspire.Hosting.Dapr/DaprSidecarResource.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Dapr;
+
+/// <summary>
+/// Represents a Dapr sidecar resource.
+/// </summary>
+internal sealed class DaprSidecarResource : Resource, IDaprSidecarResource
```

Comment:
```suggestion
internal sealed class DaprSidecarResource(string name) : Resource(name), IDaprSidecarResource { }
```

Welcome to the future